intake_solr/__init__.py

```diff
@@ -1,7 +1,4 @@
 from ._version import get_versions
 __version__ = get_versions()['version']
 del get_versions
-
-import intake  # Import this first to avoid circular imports during discovery.
-del intake
```

Comment:
I got exceptions when running with dask distributed. Removing these lines fixed the issue.

Reply:
With the move to entrypoints to declare the drivers, I hope this is no longer needed